RecCalorimeter/src/components/ReadNoiseFromFileTool.cpp

```diff
@@ -234,12 +233,12 @@ double ReadNoiseFromFileTool::getNoiseOffsetPerCell(uint64_t aCellId) {
   }
 
   // Total noise: electronics noise + pileup
-  double totalNoise = sqrt(pow(elecNoise, 2) + pow(pileupNoise, 2)) * m_scaleFactor;
+  double totalNoiseOffset = sqrt(elecNoiseOffset*elecNoiseOffset + pileupNoiseOffset*pileupNoiseOffset) * m_scaleFactor; // shouldnt the offset be summed linearly?
 
-  if (totalNoise < 1e-6) {
-    warning() << "Zero noise: cell eta " << cellEta << " layer " << cellLayer << " noise " << totalNoise << endmsg;
-  }
+  // No warning: offset is usually zero or close
+  //if (totalNoiseOffset < 1e-6) {
+  //  warning() << "Zero noise offset: cell eta " << cellEta << " layer " << cellLayer << " noise " << totalNoiseOffset << endmsg;
+  //}
```

Comment:
Can you remove this comment out part? And put comment about the offset value to line `236`?